src/sentry/models/project.py

```diff
@@ -407,10 +408,23 @@ def option_manager(self) -> ProjectOptionManager:
 
         return ProjectOption.objects
 
+    @property
+    def template_manager(self) -> ProjectTemplateOptionManager:
+        from sentry.models.options.project_template_option import ProjectTemplateOption
+
+        return ProjectTemplateOption.objects
+
     def get_option(
         self, key: str, default: Any | None = None, validate: Callable[[object], bool] | None = None
     ) -> Any:
-        return self.option_manager.get_value(self, key, default, validate)
+        # don't pass the default, we need to check the template if it doesn't exist
+        option = self.option_manager.get_value(self, key, None, validate)
+
+        if option is None and self.template is not None:
+            return self.template_manager.get_value(self.template, key, default, validate)
+
+        # check if the option is set, if not return the default
+        return default if option is None else option
```

Comment:
Does this need to be `return self.option_manager.get_value(self, key, default, validate) if option is None else option`?

Reply:
yeah, that probably would have fixed the test 🤣 i ended up making it a littttle different and just stored the result in `option` then if it's the `default` value, check the template. so logically it works the same in this function, but it will correctly utilize the `default` / `vaildator` stuff (that your recommendation would have also fixed)